modules/ROOT/pages/configure-host.adoc

```diff
@@ -31,12 +31,30 @@ For example, depending on your setup, select one of the following: AS2, DUNS num
 
 . Click *New* within the *Certificates* section to begin to upload any needed digital certificates (such as an X.509 certificate in PKCS12 file format for the AS2 transport protocol).
 
-. Click *Choose file* to select the certificate (typically a `.PEM` file), and click *Upload certificate* to upload the chosen digital certificate file. See xref:Certificates.adoc[Certificates] for more info on configuring certificates in Partner Manager.
+. Click *Choose file* to select the certificate (typically a `.PEM` file), and click *Upload certificate* to upload your chosen certificate file. See xref:Certificates.adoc[Certificates] for more info on configuring certificates in Partner Manager.
++
+NOTE: If a single certificate is uploaded, Partner Manager prevents you from deleting your only certificate.
```

Comment:
Does this really belong at this location? Is the user deleting or planning to delete a certificate at this point?

Reply:
Fixed.